x/ccv/provider/keeper/msg_server.go

```diff
@@ -70,19 +70,30 @@ func (k msgServer) AssignConsumerKey(goCtx context.Context, msg *types.MsgAssign
 	if err := k.Keeper.AssignConsumerKey(ctx, msg.ConsumerId, validator, consumerTMPublicKey); err != nil {
 		return nil, err
 	}
-	k.Logger(ctx).Info("assigned consumer key",
+
+	chainId, err := k.GetConsumerChainId(ctx, msg.ConsumerId)
+	if err != nil {
+		return nil, errorsmod.Wrapf(ccvtypes.ErrInvalidConsumerState, "cannot get consumer chain ID: %s", err.Error())
+	}
+
+	k.Logger(ctx).Info("validator assigned consumer key",
 		"consumer id", msg.ConsumerId,
+		"chain id", chainId,
```

Comment:
nit: discrepancy with what we have in previous log statements (e.g., `"chainId"` versus `"chain id"`)